### ISSUE TITLE
A problem on Tanks&Templs

### ISSUE THREAD

**Issue by crazydiamondaz** [doubleZ0108/GeoMVSNet#13]
Thanks to your great work! I used the model you provided([DTU pretrained model](https://drive.google.com/file/d/147_UbjE87E-HB9sZ5yLDbckynH825nJd/view?usp=sharing)). And run bash scripts/tnt/test_tnt.sh, scripts/tnt/fusion_tnt.sh. The results program output are noisy and not clear enough. I've upload the picture below. Could you tell me why the result is not ideal? Thanks again!
![image](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/a16ce565-8b9d-4331-b555-ea719b019ca4)
![image](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/ec939dce-5a7b-4674-806d-8832c16062be)

**Comment by crazydiamondaz** [comment]
I find an another question. After I run scripts/tnt/test_tnt.sh, scripts/tnt/fusion_tnt.sh. I found the output depth_est in Temple folder   is ill. Some depth masks are blank or show something abnormal. I've uploaded some depth maps of Temple below:
![00000018](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/7c590371-1f0f-4647-bdaa-742fe3f0b3f3)
![00000019](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/000bd0f6-0967-4d87-8021-492f702278ee)
![00000020](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/3484b5fb-fdd6-4015-860c-c1639a262dce)
![00000021](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/3310b5ba-0953-4b7b-8228-57b24c148287)
The number of these four pictures is 18-21. And the ply of Temple is also totally wrong just as below:
![image](https://github.com/doubleZ0108/GeoMVSNet/assets/46803604/76a7ca58-4ba1-4e5e-8c75-318155609dcc)
Thanks for your work again.

**Comment by doubleZ0108** [comment]
Sorry for not responding in a timely manner while on vacation recently.

The first thing you should know is that the pre-trained checkpoint model was trained on the DTU dataset which is totally different from the T&T dataset. As we mentioned, you are recommended to refine it on the BlendedMVS dataset.

As for your problem of estimated depth map. You can specify `v_min` and `v_max` when visualizing the depth map from the `PIL` library, and you can have a clear visualization of the main object of the scene. 

> Due to the large depth distribution of the scene, direct visualization will cause the depth range of the main body of the object to fall into the inappropriate range, resulting in visualization of all black or all white.

As for point cloud. We do not add any filter strategy after fusion, and I think you should do it yourself. (e.g. Statistical filtering, Spherical filtering)

**Comment by crazydiamondaz** [comment]
> Sorry for not responding in a timely manner while on vacation recently.
> 
> The first thing you should know is that the pre-trained checkpoint model was trained on the DTU dataset which is totally different from the T&T dataset. As we mentioned, you are recommended to refine it on the BlendedMVS dataset.
> 
> As for your problem of estimated depth map. You can specify `v_min` and `v_max` when visualizing the depth map from the `PIL` library, and you can have a clear visualization of the main object of the scene.
> 
> > Due to the large depth distribution of the scene, direct visualization will cause the depth range of the main body of the object to fall into the inappropriate range, resulting in visualization of all black or all white.
> 
> As for point cloud. We do not add any filter strategy after fusion, and I think you should do it yourself. (e.g. Statistical filtering, Spherical filtering)

I've understood your idea. Sorry for my carelessness and thanks to your patient and great respondence. Hope your research and work goes all well.
